spec/models/widget/card_spec.rb

```diff
@@ -45,4 +45,36 @@
       expect(Widget::Card.body).not_to include(page_card)
     end
   end
+
+  context "when is not header card" do
+    it "link_url field is required" do
+      invalid_card = build(:widget_card, header: false, link_url: nil)
+
+      expect(invalid_card).not_to be_valid
```

Comment:
```suggestion
      card = build(:widget_card, header: false, link_url: nil)

      expect(card).not_to be_valid
```

Although we've done it a couple of times, 94% of the time we don't add the `invalid_` prefix to the variable name in tests like this one :thinking:. Same applies to other `valid_` and `invalid_` prefixes in these tests. It even makes the lines shorter!